.github/workflows/install_spack.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env sh
-git clone https://github.com/spack/spack.git
-echo -e "config:\n  build_jobs: 2" > spack/etc/spack/config.yaml
-. spack/share/spack/setup-env.sh
-spack compilers
+. share/spack/setup-env.sh
+echo -e "config:\n  build_jobs: 2" > etc/spack/config.yaml
+spack compiler find
```

Comment:
I some scripts I see
```
==> Warning: apple-clang@11.0.3 cannot build optimized binaries for "ivybridge". Using best target possible: "x86_64"
```
Should we just add
```
spack config add packages:all:target:x86_64
```
(not sure if syntax is correct, it's a list argument)?